scripts/valhalla_build_timezones

```diff
@@ -1,18 +1,16 @@
 #!/bin/sh
 
 error_exit() {
-  echo "$1" 1>&2
-  #exit 1
+  echo "error: $1" 1>&2
+  exit 1
```

Comment:
This comment seems to be committed accidentally.

Reply:
Sadly it was on purpose.

Reply:
maybe we can detect the geos version and only `exit 1` if it was a newer one?

for example `pkg-config geos --modversion` returns `3.9.1` on my system and it fails because of double free but the output sqlite is perfectly fine

Reply:
interestingly on github 3.9 doesnt even exist as a tag... so perhaps thats a sign ;)

Reply:
in a month we'll have a new ubuntu version so maybe this will be rectified then..